cinema/views.py

```diff
@@ -97,25 +100,42 @@ def get_queryset(self):
     def get_serializer_class(self):
         if self.action == "list":
             return MovieListSerializer
-
         if self.action == "retrieve":
             return MovieDetailSerializer
-
+        if self.action == "upload_image":
```

Comment:
Code Style issue: Make sure you've added a blank line at the end to all your files. Although it's not a critical issue, it's a good practice to include a blank line at the end of each file. This is a common convention that helps prevent any unexpected behavior.